src/main/java/com/aws/greengrass/clientdevices/auth/api/UseCases.java

```diff
@@ -26,27 +27,13 @@ private void checkCanRun() {
     }
 
     /**
-     * Allows to provide a specific dependency we want to be injected on the use case being tested.
```

Comment:
Removing since we are not using this anymore. We determined that doing this might lead to issues when running use cases using multiple threads